WebDevStudios/wd_s
# Batch dependabot updates

Closes #959
Closes #960
Closes #961
Closes #963
Closes #964

### DESCRIPTION

Batch dependabot updates. See closed PRs for details.

### STEPS TO VERIFY

1. `gh pr checkout #`
2. `composer i && npm i &&  npm run build`
3. Verify that everything builds
4. Check other npm commands in `package.json`
5. Verify that everything looks good
